charts/kargo/templates/api/configmap.yaml

```diff
@@ -30,6 +30,9 @@ data:
   {{- end }}
   {{- if .Values.api.oidc.enabled }}
   OIDC_ENABLED: "true"
+  {{- if .Values.api.oidc.globalServiceAccounts.enabled }}
+  GLOBAL_SERVICE_ACCOUNT_NAMESPACES: {{ join "," .Values.api.oidc.globalServiceAccounts.namespaces }}
+  {{- end }}
```

Comment:
What do you think about making this unconditional? i.e. If `.Values.api.oidc.globalServiceAccounts.namespaces` has length zero, the value just ends up being the empty string.

In retrospect, I wish #1041 had done that. I'll open a separate issue or PR to fix that.